.github/workflows/backport.yml

```diff
@@ -0,0 +1,52 @@
+---
+name: Backport Assistant Runner
+
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    if: github.event.pull_request.merged
+    runs-on: ubuntu-latest
+    container: hashicorpdev/backport-assistant:0.2.3
+    steps:
+      - name: Backport changes to stable-website
+        run: |
+          backport-assistant backport -automerge
+        env:
+          BACKPORT_LABEL_REGEXP: "backport/(?P<target>website)"
+          BACKPORT_TARGET_TEMPLATE: "stable-{{.target}}"
+          GITHUB_TOKEN: ${{ secrets.ELEVATED_GITHUB_TOKEN }}
+      - name: Backport changes to latest release branch
+        run: |
+          resp=$(curl -f -s "https://api.github.com/repos/$GITHUB_REPOSITORY/releases?per_page=100")
```

Comment:
I'm guessing GITHUB_REPOSITORY resolves to `hashicorp/vault` and not just `vault`? Once I started thinking about that, I think we'll need to introduce backport-assistant to vault-enterprise as well, otherwise we'd wind up with a different workflow on OSS vs ENT.

Reply:
> I'm guessing GITHUB_REPOSITORY resolves to hashicorp/vault and not just vault?

Yep! - resolves to [`{owner}/{repo}`](https://docs.github.com/en/actions/learn-github-actions/environment-variables)

> Once I started thinking about that, I think we'll need to introduce backport-assistant to vault-enterprise as well, otherwise we'd wind up with a different workflow on OSS vs ENT.

Should I open a similar PR in https://github.com/hashicorp/vault-enterprise? (minus the `website` components)

Reply:
I'd appreciate it if you have time!

Reply:
@mladlow here's the Vault-ENT PR

I wasn't sure about the exact replacement for `backport/website` (given that it opens a backport PR against _latest_ `release/...`) so I left a TBD comment.

- https://github.com/hashicorp/vault-enterprise/pull/2361